### DISCUSSION TITLE
Iota tokens are missing (Пропали токены iota)

### DISCUSSION THREAD

**Discussion by Roma-github** [iotaledger/firefly#7315] [Help & Support]
Hi. Today I decided to transfer Iota tokens to a new version of the Firefly wallet. I bought them on the first version of the iota wallet. After entering a seed phrase of 81 characters, it shows an incorrect balance. Who can I contact to solve this problem? And how?
Привет. Сегодня решил перевести токены Iota на новую версию кошелька Firefly. Покупал их на первой версии кошелька iota wallet.  После ввода seed фразы из 81 символа, показывает не верный баланс. С кем можно связаться для решения этой проблемы?  И как?
![Снимок экрана (1)](https://github.com/iotaledger/firefly/assets/99553745/22485227-33f6-402c-b4ae-a322398e5534)

**Comment by begonaalvarezd** [comment]
Hi, it can take couple of search iterations to find your full balance, please use the button to search more until you find all your balance. I hope this helps 🙏🏼
